packages/flutter_tools/lib/src/commands/apk.dart

```diff
@@ -179,227 +189,338 @@ class ApkCommand extends FlutterCommand {
         help: 'Password for the entry within the keystore.');
   }
 
-  Future _findServices(_ApkComponents components) async {
```

Comment:
This looks like a lot of changes, but these deletions are all just moves. I moved these methods out of the ApkCommand class to be toplevel functions, so we could call them without an instance of that class.